app/questionnaire/questionnaire_schema.py

```diff
@@ -1248,6 +1268,28 @@ def get_item_anchor(self, section_id: str, list_name: str) -> str | None:
                 if item["for_list"] == list_name and item.get("item_anchor_answer_id"):
                     return f"#{str(item['item_anchor_answer_id'])}"
 
+    def _populate_placeholder_section_dependencies(self) -> None:
+        for block in self.get_blocks():
+            transforms = get_mappings_with_key("transform", block)
+            placeholder_answer_ids = {
+                item.get("identifier")
+                for transform in transforms
+                if transform["transform"] in TRANSFORMS_REQUIRING_ROUTING_PATH
+                for item in transform["arguments"]["items"]
+                if item.get("source") == "answers"
+            }
+            placeholder_dependencies = self._get_section_ids_for_answer_ids(
+                answer_ids=placeholder_answer_ids
+            )
+            if placeholder_dependencies:
+                # Type Ignore: At this point we section id cannot be None
+                section_id = self.get_section_id_for_block_id(block["id"])
+                self._placeholder_section_dependencies_by_block[section_id][  # type: ignore
+                    block["id"]
```

Comment:
Insted of type ignore this, type ignore the source of the issue, which is the line above.

Reply:
Does not seem to work when type ignoring section_id at the line above